src/test/java/com/smalaca/messagesender/repository/inmemory/InMemoryMessageRepositoryTest.java

```diff
@@ -86,4 +88,42 @@ public void shouldRecognizeThatMessageSimilarToThirdDoesNotExist() {
 
         assertFalse(repository.exists(factory.createFrom(messageDto, "123456")));
     }
+
+    @Test
+    public void shouldDeletePreviouslyCreatedNewMessage(){
+
+        MessageDto messageDto = new MessageDto();
+        messageDto.setSubject("some subject 2");
+        messageDto.setBody("some body 2");
+        messageDto.setTo("smalaca");
+        messageDto.setFrom("javakrk5");
+
+        Message message = factory.createFrom(messageDto, "777");
+        repository.add(message);
+
+        Assert.assertTrue(repository.exists("777"));
```

Comment:
to jest test metody delete. Dodawanie powinno być przetestowane w innych metodach.